port/resource_port_action.go

```diff
@@ -156,7 +170,14 @@ func actionResourceToBody(d *schema.ResourceData) (*cli.Action, error) {
 	action.Title = d.Get("title").(string)
 	action.Icon = d.Get("icon").(string)
 	action.Description = d.Get("description").(string)
-	action.InvocationMethod = d.Get("invocation_method").(string)
+	if invocation_method, ok := d.GetOk("invocation_method"); ok {
```

Comment:
```suggestion
	if invocationMethod, ok := d.GetOk("invocation_method"); ok {
```
golang variables are commonly camelCased